src/Algebra/Graph/Export/Dot.hs

```diff
@@ -34,6 +34,9 @@ import qualified Algebra.Graph.Export as E
 -- Attributes are used to specify the style of graph elements during export.
 data Attribute s = (:=) s s
 
+-- | Used to determine whether to use double-quotes around attribute values.
+data Quoting = DoubleQuote | NoQuote
```

Comment:
I weakly prefer plural `DoubleQuotes` and `NoQuotes` but I'll leave it to you to decide what's best.